cmd/sonobuoy/app/status_test.go

```diff
@@ -94,7 +95,7 @@ func TestPrintStatus(t *testing.T) {
 			}
 
 			if b.String() != test.expected {
-				t.Errorf("expected output to be %q, got %q", test.expected, b.String())
```

Comment:
Tweaked this so that each of "expected" and "got" would start on a new line; just thought it looked more clear at a glance.